README.rst

```diff
@@ -277,9 +302,9 @@ Note that RNNT requires numba to be installed from conda.
   pip uninstall numba
   conda install -c conda-forge numba
 
-NeMo Megatron
-~~~~~~~~~~~~~
-NeMo Megatron training requires NVIDIA Apex to be installed.
+Apex
+~~~~
+NeMo LLM Domain training requires NVIDIA Apex to be installed.
 Install it manually if not using the NVIDIA PyTorch container.
 
 To install Apex, run
```

Comment:
I encountered issue here, wondering if anyone verified the Apex branch/checkpoint is still compatible?

Reply:
That can be fixed with update in commit. I didn't get any commit from team, so I'll leave it as is for the moment